src/repl/helper.rs

```diff
@@ -0,0 +1,71 @@
+use rustyline::{
+    completion::{Completer, Pair},
+    error::ReadlineError,
+    highlight::{Highlighter, MatchingBracketHighlighter},
+    hint::{Hinter, HistoryHinter},
+    validate::{self, MatchingBracketValidator, Validator},
+    Context,
+};
+use rustyline_derive::Helper;
+use std::borrow::Cow;
+
+#[derive(Helper)]
+pub(super) struct ReplHelper {
+    pub(super) highlighter: MatchingBracketHighlighter,
+    pub(super) validator: MatchingBracketValidator,
+    pub(super) hinter: HistoryHinter,
+}
+
+impl Completer for ReplHelper {
+    type Candidate = Pair;
+
+    fn complete(
+        &self,
+        _line: &str,
+        _pos: usize,
+        _ctx: &Context<'_>,
+    ) -> Result<(usize, Vec<Pair>), ReadlineError> {
+        Ok((0, vec![]))
+    }
```

Comment:
I'd love to do lots of fancy things with tab-completion, but that's a larger project, no need for it in a first draft.